templates/index.html

```diff
@@ -453,438 +494,435 @@ <h2>Do more with trusted open source</h2>
-                                height="35",
-                                hi_def=True,
-                                loading="lazy") | safe
-                }}
-              </div>
-              <div class="col-9 col-medium-6">
-                <div class="p-section">
-                  <div class="row">
-                    <div class="col-9 col-medium-6">
-                      <h3 class="p-heading--4">
-                        Securing medical devices
-                        <br class="u-hide--small u-hide--medium" />
-                        to improve healthcare outcomes
-                      </h3>
-                    </div>
-                    <div class="p-image--container u-hide--large">
-                      {{ image(url="https://assets.ubuntu.com/v1/ea5462e7-3.png",
-                                            alt="",
-                                            width="2464",
-                                            height="1027",
-                                            hi_def=True,
-                                            attrs={"class": "p-image--container__image"},
-                                            loading="lazy") | safe
-                      }}
-                    </div>
-                    <div class="col-6 col-medium-6">
-                      <p>
-                        <i>“We looked at alternatives to an Ubuntu Pro subscription, but they were, without exception, far more laborious. With Ubuntu Pro, we can meet our customers’ expectations around security, because Canonical guarantees patching for all open source components that we are using.”</i>
-                      </p>
-                      <a class="p-cta-text u-hide--medium u-hide--small"
-                         href="/case-study/grundium-ubuntu-pro-for-devices"
-                         aria-label="Inspect the case study for Grundium">Inspect the case study
-                        <i class="p-icon--arrow-right"></i></a>
-                    </div>
-                    <div class="col-3 col-medium-3">
-                      <p class="u-no-margin--bottom">
-                        <b>Kimmo Alanen</b>
-                      </p>
-                      <p class="u-text--muted">
-                        Vice President,
-                        <br />
-                        Grundium
-                      </p>
-                    </div>
-                    <a class="p-cta-text u-hide--large"
-                       href="/case-study/grundium-ubuntu-pro-for-devices"
-                       aria-label="Inspect the case study for Grundium">Inspect the case study
-                      <i class="p-icon--arrow-right"></i></a>
-                  </div>
+    <div tabindex="0"
+          role="tabpanel"
+          id="enhance-security-tab"
+          aria-labelledby="enhance-security"
+          aria-hidden="false">
+      <div class="p-section">
+        <div class="row">
+          <div class="col-3" style="padding-top: 0.45rem;">
```

Comment:
This padding nudge is repeated a few times, can we move it to the stylesheet for easier maintainability?

Reply:
I see this is in production as well.
I will make the adjustment

Reply:
Fixed